### DISCUSSION TITLE
PrimeNg calendar in not showing focus border

### DISCUSSION THREAD

**Discussion by ravisit** [primefaces/community#1197] [PrimeNG] [answered]
calendar in not showing focus border when trying to navigate via the arrow keys within the dialog dates.

![image](https://github.com/primefaces/community/assets/29448663/53be1aff-bab1-4085-aa18-f57acd716287)

**Comment by cetincakiroglu** [comment]
Hi,

Could you please provide a stackblitz example so we can identify the issue?

**Reply by ravisit** [reply, marked as answer]
This issue is resolved now. Thanks
